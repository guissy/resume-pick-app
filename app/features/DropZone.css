.mainBox {
  flex: 1;
  min-height: 90px;
  margin-top: 20px;
  border: 1px dashed #ddd;
  cursor: pointer;
}
.dropBox {
  width: 100%;
  height: 100%;
  padding: 30px;
  display: flex;
  justify-content: center;
  align-content: center;
  box-sizing: border-box;
}

.placeholder {
  width: 100%;
  color: #eee;
}

.textCenter {
  text-align: center;
}

.upload {
  color: #666;
}

.mainBox:hover .upload {
  color: #aaa;
}
