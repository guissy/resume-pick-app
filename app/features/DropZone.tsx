import React from 'react';
// Import the useDropzone hooks from react-dropzone
import { DropzoneProps, useDropzone } from 'react-dropzone';
import styles from './DropZone.css';

const DropZone = ({ onDrop, accept }: Partial<DropzoneProps>) => {
  // Initializing useDropzone hooks with options
  const { getRootProps, getInputProps, isDragActive } = useDropzone({
    onDrop,
    accept,
  });

  /*
    useDropzone hooks exposes two functions called getRootProps and getInputProps
    and also exposes isDragActive boolean
  */

  return (
    // eslint-disable-next-line react/jsx-props-no-spreading
    <div {...getRootProps()} className={styles.mainBox}>
      {/* eslint-disable-next-line react/jsx-props-no-spreading */}
      <input className="dropzone-input" {...getInputProps()} />
      <div className={styles.dropBox}>
        <section className={styles.placeholder}>
          {isDragActive ? (
            <p className={styles.textCenter}>文件放在这里</p>
          ) : (
            <p className={styles.textCenter}>
              {/* eslint-disable-next-line react/no-unescaped-entities */}
              拖拽文件到这里
            </p>
          )}
        </section>
      </div>
    </div>
  );
};

export default DropZone;
