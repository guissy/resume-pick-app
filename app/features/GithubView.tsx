import React from 'react';
import { GhOpt, HTMLWebview } from './type';
import styles from './ScoreList.css';

type Props = {
  url: string;
};
const GithubView: React.FC<Props> = ({ url }) => {
  const webviewRef = React.useRef<HTMLWebview | null>(null);
  const [repos, setRepos] = React.useState(NaN);
  const [contrib, setContrib] = React.useState(NaN);
  const [followers, setFollowers] = React.useState(NaN);
  // const [calendar, setCalendar] = React.useState('');
  React.useEffect(() => {
    webviewRef.current?.addEventListener('dom-ready', () => {
      webviewRef.current
        ?.executeJavaScript(
          `JSON.stringify({
            repos: parseInt(document.querySelector('.Counter')?.textContent),
            followers: parseInt(document.querySelector(".js-profile-editable-area a")?.textContent?.trim()),
            contrib: parseInt(document.querySelector('.js-yearly-contributions h2')?.textContent?.trim()),
            calendar: "document.querySelector('.js-calendar-graph-svg').outerHTML",
          })`,
          false
        )
        .then((data) => {
          const result = JSON.parse(data) as GhOpt;
          if (typeof result.repos === 'number') {
            setRepos(result.repos);
          }
          if (typeof result.contrib === 'number') {
            setContrib(result.contrib);
          }
          if (typeof result.followers === 'number') {
            setFollowers(result.followers);
          }
          // setCalendar(result.calendar);
        });
    });
  }, [webviewRef]);
  const tail = url.split('://').pop();
  const [host, path] = tail?.split('/') || [];
  const urlOk = `https://${host}/${path.split('?').shift()}`;
  return url.includes('github') ? (
    <div>
      <webview ref={webviewRef} src={urlOk} style={{ display: 'none' }} />
      {repos >= 0 && (
        <>
          <span className={styles.subName}>repo</span>
          <span className={styles.gained}>{repos}</span>
          <br />
        </>
      )}
      {followers >= 0 && (
        <>
          <span className={styles.subName}>followers</span>
          <span className={styles.gained}>{followers}</span>
          <br />
        </>
      )}
      {contrib >= 0 && (
        <>
          <span className={styles.subName}>contrib</span>
          <span className={styles.gained}>{contrib}</span>
        </>
      )}
      {/* <br /> */}
      {/* <span dangerouslySetInnerHTML={{ __html: calendar }} /> */}
    </div>
  ) : (
    <></>
  );
};

export default GithubView;
