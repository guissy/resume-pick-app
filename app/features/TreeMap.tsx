import React from 'react';
import * as d3 from 'd3';
import { HierarchyRectangularNode } from 'd3';
import capitalize from 'lodash/capitalize';
import { useSelector } from 'react-redux';
import { Keyword, ScoreFile } from './type';
import { selectSearch } from './scoreSlice';

type Props = {
  scoreFile: ScoreFile;
  index: number;
};
const width = 360;
const height = 120;

const TreeMap: React.FC<Props> = ({ scoreFile, index }) => {
  const ref = React.useRef(null);
  const search = useSelector(selectSearch);
  const key = React.useMemo(() => {
    if (scoreFile?.keywords?.length > 0) {
      return (Date.now() / 5000).toFixed(0);
    }
    return '';
  }, [scoreFile?.keywords]);
  React.useEffect(() => {
    if (ref.current) {
      // setTimeout(() => {
      const totalGained = scoreFile.keywords
        .map((v) => v.gained)
        .reduce((s, v) => s + v, 0);
      const widthOk = Math.max(Math.min((width * totalGained) / 20, 360), 80);
      const svg = d3.select(ref.current);
      const treeMap = d3
        .treemap()
        .tile(d3.treemapSquarify)
        .size([widthOk, height])
        .round(true)
        .paddingInner(1);
      const data = {
        name: '',
        children: scoreFile.keywords
          .filter((k) => !!k.children.length)
          .map((k) => ({
            name: k.name,
            children: k.children
              .filter((w) => w.gained > 0)
              .map((w) => ({
                name: w.name,
                gained: w.gained,
              })),
          })),
        gained: NaN,
      };
      const hi = d3
        .hierarchy(data)
        .sum((d) => d.gained)
        .sort(
          (a, b) => (b.value || 0) - (a.value || 0)
        ) as HierarchyRectangularNode<Keyword>;

      treeMap(hi);

      const cell = svg.selectAll('g').data(hi.leaves()).enter().append('g');

      cell
        .append('rect')
        .attr('x', (d) => d.x0)
        .attr('y', (d) => d.y0)
        .attr('width', (d) => d.x1 - d.x0)
        .attr('height', (d) => d.y1 - d.y0)
        .attr('fill', (d) => {
          const r = Math.min(d.data.gained + 0.5 / 4.5, 0.9);
          const alpha = Math.round(r * 255).toString(16);
          if (
            search.toLowerCase().split(' ').includes(d.data.name.toLowerCase())
          ) {
            return `#ff0000${alpha}`;
          }
          return `#0000ff${alpha}`;
        });

      cell
        .append('text')
        .attr('x', (d) => d.x0)
        .attr('y', (d) => d.y0)
        .attr('fill', (d) => {
          const r = Math.min(((d.x1 - d.x0) * (d.y1 - d.y0)) / 1200, 1);
          const alpha = Math.round(r * 255).toString(16);
          return `#ffffff${alpha}`;
        })
        .attr('font-size', (d) => {
          // eslint-disable-next-line no-control-regex
          const isZh = d.data.name.match(/[^\x00-\xff]/);
          const len = Math.max(d.data.name.length, 2.5);
          const w0 = d.x1 - d.x0;
          const w = (w0 / len) * 1.45;
          const h = d.y1 - d.y0;
          return Math.min(Math.min(isZh ? w * 0.6 : w, h), 36).toFixed(1);
        })
        .attr('dx', function getDx(d) {
          // eslint-disable-next-line react/no-this-in-sfc
          const fontSize = parseFloat(this.getAttribute('font-size') || '12');
          const w0 = d.x1 - d.x0;
          return (w0 - fontSize * 0.72 * d.data.name.length ** 0.88) / 2;
        })
        .attr('dy', function getDy(d) {
          const h = d.y1 - d.y0;
          // eslint-disable-next-line react/no-this-in-sfc
          const fontSize = parseFloat(this.getAttribute('font-size') || '12');
          const y = h - Math.max((h - fontSize) / 2, 0) - fontSize / 10;
          return y.toFixed(1);
        })
        .text((d) => {
          const { name } = d.data;
          return name.length > 3 ? capitalize(name) : name.toUpperCase();
        });
    }
  }, [ref, scoreFile?.keywords, index, search]);
  return (
    <div>
      <svg key={key} ref={ref} width={width} height={height} />
    </div>
  );
};

export default TreeMap;
