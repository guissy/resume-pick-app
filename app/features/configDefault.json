[
  {
    "name": "核心项",
    "children": [
      {
        "name": "RN",
        "alias": ["react native", "rn", "weex"],
        "score": 1.5
      },
      {
        "name": "flutter",
        "alias": ["flutter", "dart"],
        "score": 1.5
      },
      {
        "name": "h5",
        "alias": [
          "webview",
          "uni-app",
          "uniapp",
          "umi-app",
          "ionic",
          "apicloud",
          "mobile",
          "hybird"
        ],
        "score": 1.5
      },
      {
        "name": "react",
        "alias": ["redux", "antd", "ant design"],
        "score": 1.5
      },
      {
        "name": "vue",
        "alias": ["vue2", "vuejs", "vue2", "element", "vant"],
        "score": 1.5
      },
      {
        "name": "vue3",
        "score": 1.5
      },
      {
        "name": "angular",
        "alias": [
          "ng",
          "ng2",
          "angular2",
          "angular4",
          "angular5",
          "angular6",
          "angular7",
          "angular8"
        ],
        "score": 1
      },
      {
        "name": "wxapp",
        "alias": ["微信", "小程序", "taro", "mpvue", "mpvue"],
        "score": 1.5
      },
      {
        "name": "node",
        "alias": ["nodejs", "express", "keo", "egg", "next", "typeorm", "pm2"],
        "score": 1.5
      },
      {
        "name": "electron",
        "alias": ["nwjs"],
        "score": 1.5
      },
      {
        "name": "state mgr",
        "alias": [
          "redux",
          "saga",
          "mobx",
          "vuex",
          "vuetify",
          "dva",
          "事件总线",
          "eventBus",
          "event bus"
        ],
        "score": 1
      },
      {
        "name": "es6",
        "alias": [
          "es5",
          "es6+",
          "es2015",
          "es2016",
          "es2017",
          "es7",
          "es8",
          "es9",
          "ecmascript"
        ],
        "score": 0.5
      },
      {
        "name": "w3c",
        "score": 0.5,
        "alias": ["mdn"]
      },
      {
        "name": "css3",
        "alias": [
          "flex",
          "grid",
          "css3",
          "rem",
          "less",
          "sass",
          "css-modules",
          "css in js",
          "css-in-js",
          "styled-components",
          "jss",
          "emotion"
        ],
        "score": 0.5
      }
    ]
  },
  {
    "name": "进阶",
    "children": [
      {
        "name": "typescript",
        "alias": ["ts"],
        "score": 1.5
      },
      {
        "name": "rxjs",
        "alias": ["ReactiveX", "ngrx"],
        "score": 1.5
      },
      {
        "name": "graphql",
        "alias": ["gql"],
        "score": 1.5
      },
      {
        "name": "test",
        "calc": "max",
        "alias": [
          "mocha",
          "jasmine",
          "jest",
          "cypress",
          "puppeteer",
          "unit test"
        ],
        "score": 1.5
      },
      {
        "name": "lint",
        "alias": ["tslint", "eslint", "jslint", "命名规范", "代码规范"],
        "score": 0.5
      },
      {
        "name": "review",
        "alias": ["pr", "git flow", "代码审查"],
        "score": 0.5
      },
      {
        "name": "gayhub",
        "alias": ["github.com", "gitee.com"],
        "score": 1.5
      },
      {
        "name": "blog",
        "alias": [
          "个人博客",
          "博客地址",
          "csdn.net",
          "cnblogs.com",
          "juejin.im",
          "zhihu.com",
          "jianshu.com",
          "segmentfault.com",
          "oschina.net",
          "imooc.com",
          "51cto.com",
          "gitbooks.io"
        ],
        "score": 1.5
      }
    ]
  },
  {
    "name": "其他",
    "children": [
      {
        "name": "leader",
        "alias": ["组长", "负责人"],
        "score": 1
      },
      {
        "name": "mock",
        "score": 0.5
      },
      {
        "name": "git",
        "score": 0.5
      },
      {
        "name": "docker",
        "score": 0.5
      },
      {
        "name": "websocket",
        "alias": ["socket", "canvas", "charles"],
        "score": 0.5
      },
      {
        "name": "微服务",
        "score": 0.5
      },
      {
        "name": "加密",
        "alias": ["密文", "支付", "md5", "sha1", "签名", "白名单"],
        "score": 0.5
      },
      {
        "name": "重构",
        "alias": ["防抖", "去抖", "节流", "虚拟化", "懒加载"],
        "score": 0.5
      },
      {
        "name": "自定义",
        "alias": ["封装", "公共组件", "二次封装"],
        "score": 0.5
      },
      {
        "name": "python",
        "score": 0.5
      },
      {
        "name": "java",
        "score": 0.5
      },
      {
        "name": "go",
        "score": 0.5
      },
      {
        "name": "rust",
        "score": 0.5
      },
      {
        "name": "cpp",
        "score": 0.5
      },
      {
        "name": "swift",
        "score": 0.5
      },
      {
        "name": "kotlin",
        "score": 0.5
      },
      {
        "name": "scala",
        "score": 0.5
      },
      {
        "name": "mysql",
        "score": 0.5
      },
      {
        "name": "mongodb",
        "score": 0.5
      },
      {
        "name": "electron",
        "score": 0.5
      },
      {
        "name": "devops",
        "alias": ["shell", "nginx", "linux", "cicd"],
        "score": 0.5
      },
      {
        "name": "vim",
        "score": 0.5
      },
      {
        "name": "ssr",
        "score": 0.5
      },
      {
        "name": "bi",
        "score": 0.5,
        "alias": ["可视化", "大屏", "echarts", "d3"]
      },
      {
        "name": "three.js",
        "score": 0.5
      },
      {
        "name": "postman",
        "score": 0.5,
        "alias": ["charles", "fiddler"]
      },
      {
        "name": "leetcode",
        "score": 0.5,
        "alias": ["力扣", "leet code"]
      },
      {
        "name": "deno",
        "score": 0.5
      },
      {
        "name": "wasm",
        "score": 0.5,
        "alias": ["web assembly"]
      },
      {
        "name": "iframe",
        "alias": ["flash", "dreamweaver", "div+css", "v-for", "v-if"],
        "score": -1.5
      },
      {
        "name": "ajax",
        "alias": ["ajax", "jsonp", "ie6", "ie7", "ie8"],
        "score": -1
      },
      {
        "name": "jquery",
        "alias": ["jq", "zepto"],
        "score": -0.5
      },
      {
        "name": "svn",
        "alias": ["requirejs", "gulp", "grunt", "svn"],
        "score": -1
      },
      {
        "name": "积极乐观",
        "alias": ["为人和蔼", "吃苦耐劳", "上进", "了解"],
        "score": -0.5
      },
      {
        "name": "熟悉",
        "score": -0.3
      }
    ]
  }
]
