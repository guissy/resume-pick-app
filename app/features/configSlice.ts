import { createSlice } from '@reduxjs/toolkit';
// eslint-disable-next-line import/no-cycle
import { AppThunk, RootState } from '../store';
import { ConfigFile } from './type';
import initFile, { initDefaultFile, saveFile } from './configUtil';

export const configSlice = createSlice({
  name: 'config',
  initialState: { configs: {} as ConfigFile },
  reducers: {
    buildConfig: (state, action) => {
      if (Object.keys(state.configs).length === 0) {
        state.configs = action.payload;
      }
    },
    updateConfig: (state, action) => {
      Object.assign(state.configs, action.payload);
    },
    saveConfig: (state) => {
      saveFile(state.configs);
    },
    resetConfig: (state, action) => {
      state.configs = action.payload;
      saveFile(state.configs);
    },
  },
});

export const {
  updateConfig,
  buildConfig,
  saveConfig,
  resetConfig,
} = configSlice.actions;

export const initConfigAsync = (): AppThunk => (dispatch) =>
  initFile()
    .then((data) => dispatch(buildConfig(data)))
    // eslint-disable-next-line no-console
    .catch((e) => console.error(e));

export const resetConfigAsync = (): AppThunk => (dispatch) =>
  initDefaultFile()
    .then((data) => dispatch(resetConfig(data)))
    // eslint-disable-next-line no-console
    .catch((e) => console.error(e));

export default configSlice.reducer;

export const selectConfig = (state: RootState) => state.config.configs.default;
