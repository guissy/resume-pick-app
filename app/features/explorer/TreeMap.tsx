import React from 'react';
import * as d3 from 'd3';
import { HierarchyRectangularNode } from 'd3';
import capitalize from 'lodash/capitalize';
import { Keyword, ScoreFile } from '../type';

type Props = {
  scoreFile: ScoreFile;
  search: string;
};
const width = 360;
const height = 120;

const TreeMap: React.FC<Props> = ({ scoreFile, search }) => {
  const ref = React.useRef(null);
  const key = React.useMemo(() => {
    if (scoreFile?.keywords?.length > 0) {
      return (Date.now() / 5000).toFixed(0);
    }
    return '';
  }, [scoreFile?.keywords]);
  React.useEffect(() => {
    if (ref.current) {
      const searchs = (search || '').trim().split(' ');
      const keywords = scoreFile?.keywords
        .filter((k) => !!k.children.length)
        .map((k) => ({
          ...k,
          children: k.children.filter(
            (w) => k.name === 'search' || !searchs.includes(w.name)
          ),
        }));
      const totalGained = keywords
        .map((v) => v.gained)
        .reduce((s, v) => s + v, 0);
      const widthOk = Math.max(Math.min((width * totalGained) / 20, 360), 80);
      const svg = d3.select(ref.current);
      const treeMap = d3
        .treemap()
        .tile(d3.treemapSquarify)
        .size([widthOk, height])
        .round(true)
        .paddingInner(1);
      const data = {
        name: '',
        children: keywords.map((k) => ({
          name: k.name,
          children: k.children
            .filter((w) => w.gained > 0)
            .map((w) => ({
              name: w.name,
              gained: w.gained,
            })),
        })),
        gained: NaN,
      };
      const hi = d3
        .hierarchy(data)
        .sum((d) => d.gained)
        .sort(
          (a, b) => (b.value || 0) - (a.value || 0)
        ) as HierarchyRectangularNode<Keyword>;

      treeMap(hi);

      const cell = svg.selectAll('g').data(hi.leaves()).enter().append('g');

      cell
        .append('rect')
        .attr('x', (d) => d.x0)
        .attr('y', (d) => d.y0)
        .attr('width', (d) => d.x1 - d.x0)
        .attr('height', (d) => d.y1 - d.y0)
        .attr('fill', (d) => {
          const r = Math.min(d.data.gained + 0.5 / 4.5, 0.9);
          const alpha = Math.round(r * 255).toString(16);
          if (
            search.toLowerCase().split(' ').includes(d.data.name.toLowerCase())
          ) {
            return `#ff0000${alpha}`;
          }
          return `#0000ff${alpha}`;
        });

      cell
        .append('text')
        .attr('x', (d) => d.x0)
        .attr('y', (d) => d.y0)
        .attr('fill', (d) => {
          const r = Math.min(((d.x1 - d.x0) * (d.y1 - d.y0)) / 1200, 1);
          const alpha = Math.round(r * 255).toString(16);
          return `#dddddd${alpha}`;
        })
        .attr('font-size', (d) => {
          // eslint-disable-next-line no-control-regex
          const isZh = d.data.name.match(/[^\x00-\xff]/);
          const len = Math.max(d.data.name.length, 2.5);
          const w0 = d.x1 - d.x0;
          const w = (w0 / len) * 1.45;
          const h = d.y1 - d.y0;
          return Math.min(Math.min(isZh ? w * 0.6 : w, h), 36).toFixed(1);
        })
        .attr('dx', function getDx(d) {
          // eslint-disable-next-line react/no-this-in-sfc
          const fontSize = parseFloat(this.getAttribute('font-size') || '12');
          const w0 = d.x1 - d.x0;
          // eslint-disable-next-line no-control-regex
          const isZh = d.data.name.match(/[^\x00-\xff]/);
          return (
            ((w0 - fontSize * 0.72 * d.data.name.length ** 0.88) / 2) *
            (isZh ? 0.5 : 1)
          );
        })
        .attr('dy', function getDy(d) {
          const h = d.y1 - d.y0;
          // eslint-disable-next-line react/no-this-in-sfc
          const fontSize = parseFloat(this.getAttribute('font-size') || '12');
          const y = h - Math.max((h - fontSize) / 2, 0) - fontSize / 10;
          return y.toFixed(1);
        })
        .text((d) => {
          const { name } = d.data;
          return name.length > 3 ? capitalize(name) : name.toUpperCase();
        });
    }
  }, [ref, scoreFile?.keywords, search]);
  return (
    <div>
      <svg key={key} ref={ref} width={width} height={height} />
    </div>
  );
};

export default TreeMap;
