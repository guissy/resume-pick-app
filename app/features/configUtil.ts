// eslint-disable-next-line @typescript-eslint/ban-ts-comment
// @ts-ignore
import storage from 'electron-storage';
import configDefault from './configDefault.json';
import { Config } from './type';

function initFile(): Promise<Record<string, Config>> {
  return storage.get('keywords.json').catch((err: unknown) => {
    // eslint-disable-next-line no-console
    console.error(err);
    return storage.set('keywords.json', { default: configDefault });
  });
}

export function initDefaultFile(): Promise<Record<string, Config>> {
  // eslint-disable-next-line @typescript-eslint/ban-ts-comment
  // @ts-ignore
  return Promise.resolve({ default: configDefault });
}

export function saveFile(configs: Record<string, Config>) {
  return storage
    .set('keywords.json', JSON.stringify(configs, null, '  '))
    .catch((err: unknown) => {
      // eslint-disable-next-line no-console
      console.error(err);
    });
}

export default initFile;
