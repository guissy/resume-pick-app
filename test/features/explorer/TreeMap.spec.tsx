import React from 'react';
import Enzyme from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
import renderer from 'react-test-renderer';
import TreeMap from '../../../app/features/explorer/TreeMap';
import mockScoreFile from '../utils/mockScoreFile';
import { ScoreFile } from '../../../app/features/type';

Enzyme.configure({ adapter: new Adapter() });

describe('TreeMap component', () => {
  it('should match exact snapshot', () => {
    const tree = renderer
      .create(
        <TreeMap scoreFile={(mockScoreFile[0] as unknown) as ScoreFile} />
      )
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
