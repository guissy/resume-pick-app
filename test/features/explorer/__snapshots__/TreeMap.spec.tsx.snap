// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TreeMap component should match exact snapshot 1`] = `
<div>
  <svg
    height={120}
    width={360}
  />
</div>
`;
